server/bin/pbench-unpack-tarballs.sh

```diff
@@ -136,7 +136,8 @@ function  move_symlink() {
     mv ${ARCHIVE}/${hostname}/${linksrc}/${resultname}.tar.xz ${ARCHIVE}/${hostname}/${linkdest}/${resultname}.tar.xz
     status=${?}
     if [[ ${status} -ne 0 ]]; then
-        echo "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}" | tee -a ${mail_content} >&4
+        log_error "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}"
+        status_append "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}" ${mail_content}
```

Comment:
Perhaps `status_append` could also issue the log so that you don't repeat code unnecessarily?

Reply:
There are also some messages that logs only as log_error like:
https://github.com/distributed-system-analysis/pbench/blob/40b5afc64fe3391b45a558be062988dae4660e9c/server/bin/pbench-server-prep-shim-002.sh#L106

and there are messages that logs only in status_update like:
https://github.com/distributed-system-analysis/pbench/blob/40b5afc64fe3391b45a558be062988dae4660e9c/server/bin/pbench-server-prep-shim-002.sh#L200

Reply:
ok